monkey/infection_monkey/transport/attack_telems/victim_host_telem.py

```diff
@@ -14,5 +14,5 @@ def __init__(self, technique, status, machine, data=None):
         :param data: Other data relevant to the attack technique
         """
         super(VictimHostTelem, self).__init__(technique, status, data)
-        victim_host = {'hostname': machine.domain_name, 'ip': machine.ip_addr}
+        victim_host = {'domain_name': machine.domain_name, 'ip_addr': machine.ip_addr}
```

Comment:
Domain name != hostname. It should remain hostname!

Reply:
Well, VictimHost has a `domain_name` property, not `hostname`. I changed it for consistency.